src/Nettle.jl

```diff
@@ -23,10 +26,11 @@ end
 
 function __init__()
     global const nettle_major_version = get_libnettle_version()
-    hash_init()
-    cipher_init()
 end
 
-# similar to Python's hmac.HMAC.hexdigest
-hexdigest!(state::Union(HMACState,HashState)) = bytes2hex(digest!(state))
+if VERSION >= v"0.4.0-dev+6521"
+    include("precompile.jl")
+    _precompile_()
```

Comment:
This is the `SnoopCompile` generated stuff; the actual `__precompile__()` stuff is at the top of this file.